frontend/package.json

```diff
@@ -29,6 +29,7 @@
     "bootstrap": "^5.3.3",
     "eslint-config-airbnb-base": "^15.0.0",
     "formik": "^2.4.6",
+    "prop-types": "^15.8.1",
```

Comment:
On Stage 1 we discussed using prop-types. @LocalNewsTV also complimented this on this PR. I didn't know it and went to do some research about. I realised you guys are right and we should use it. Let's add a cleanup ticket for refactoring the prop type definitions we have to use prop-types instead.

Reply:
Also, when and why did we add Bootstrap to the project? Are we using it anywhere? Could be another cleanup. Just noticed it now because I was checking if prop-types was in 'dependencies', not 'devDependencies'.
Should we also add 'remove bootstrap' to our cleaning list?

Reply:
@GDamaso 
I definitely didn't recommend an additional package for prop types. 
I recommended making a type, and just calling it `PropTypes`

```ts
type PropTypes = {
    componentProp: string
}

const MyComponent = ({componentProp}: PropTypes) => {
   // Component Logic
}
```
Since you dont export the `PropTypes` type, you can just reuse the name across files. :)

Reply:
Yup in this case it helps me define a function type. I'm not sure if types can do the same thing, but I'm gonna be honest. I haven't tried it yet!

Reply:
```ts
Type function = {
    noReturnOrParam: () => void;
    returnBoolNoParam: () => boolean;
    twoParamNoReturn: (a: string, b: number) => void
}

etc, etc

Reply:
That solves it. 😝

Reply:
Another option for functions is:
```typescript
Type functionTypes = {
    noReturnOrParam(): void,
    returnBoolNoParam():  boolean,
    twoParamNoReturn(a: string, b: number): void,
}

etc, etc
```

I've used this form before, in case you see it :smile: